rov/movement/controls/PID_Controller.py

```diff
@@ -1,18 +1,19 @@
 import numpy
 
-
 class PID(object):
     """A generic PID loop controller which can be inherited and used in other control algorithms"""
 
     def __init__(self, startingError):
```

Comment:
I know this was already committed, but this might be nicer to have the constructor defined as:
```
def __init__(self, startingError, p=1, i=0, d=0)
```

So when you want to use it it's:
```
pid = PID(0, p=1)
# or
pid = PID(0, p=1, i=1)
```
instead of
```
pid = PID(0)
pid.p = 1
pid.i = 1
...
```